packages/compass-components/src/components/editor.tsx

```diff
@@ -75,10 +77,21 @@ function Editor({
     ...(!!completer && { enableLiveAutocompletion: true }),
   };
 
+  const editorRef = useRef<AceEditor | null>(null);
+
+  useEffect(() => {
```

Comment:
This probably can be a layout effect so that the id is assigned before anything is even flushed to the dom

Reply:
Good call, updated